frigate/config.py

```diff
@@ -775,6 +785,7 @@ def runtime_config(self) -> FrigateConfig:
         # Global config to propegate down to camera level
         global_config = config.dict(
             include={
+                "birdseye": ...,
```

Comment:
If you disable globally and then enable a single camera, does that enable Birdseye for the one camera?

Reply:
With birdseye disabled globally, there is no option in the sidebar. If I bypass that then it still isn't enabled as the global check to add the converter["birdseye"] doesn't go through. Not sure what the best behavior would be; I could add a warning in the config if it is disabled globally, or adapt those behaviors to consider the camera level as well, or perhaps it is good as is.